gcloud/datastore/test_client.py

```diff
@@ -18,29 +18,44 @@
 class TestClient(unittest2.TestCase):
 
     DATASET_ID = 'DATASET'
+    CONNECTION = object()
```

Comment:
@tseaver I think you meant to make that comment below about `unittest2` and interned values, checking `is` vs. `==`.